ramls/mod-users-bl.raml

```diff
@@ -218,6 +219,43 @@ resourceTypes:
           body:
             text/plain:
               example: "Internal server error"
+  /login-with-expiry:
```

Comment:
The ModuleDescriptor should be adopted accordingly.

Reply:
Added. See what you think.